### PR TITLE
Fix broken test

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit 6bc38df736526b5fc5a1052cc95844ef513d656e with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55002/6bc38df736526b5fc5a1052cc95844ef513d656e/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>✅ success</td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>⏳ pending</td></tr>
<tbody>
</table>

**Comment by CheSema** [comment]
Is that bad that database replicated tests use the same path?

**Comment by kssenii** [comment]
> Is that bad that database replicated tests use the same path?

Yes, a cache key is a hash of the s3 blob path, so if two servers read the same data, they will have the same cache keys in cache, so if one of the servers decides to delete this key, it will remove it from fs, but another server will think that this cache key is present on fs while it is already not there.

**Comment by CheSema** [comment]
Than I do not understand why `/var/lib/clickhouse/filesystem_caches_1` is used for server1 and server2. Should it be different pathes also?

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -69,6 +69,16 @@ else
 fi
 
 if [[ -n "$USE_DATABASE_REPLICATED" ]] && [[ "$USE_DATABASE_REPLICATED" -eq 1 ]]; then
+    sudo cat /etc/clickhouse-server1/config.d/filesystem_caches_path.xml \
+    | sed "s|<filesystem_caches_path>/var/lib/clickhouse/filesystem_caches/</filesystem_caches_path>|<filesystem_caches_path>/var/lib/clickhouse/filesystem_caches_1/</filesystem_caches_path>|" \
+    > /etc/clickhouse-server1/config.d/filesystem_caches_path.xml.tmp
+    mv /etc/clickhouse-server1/config.d/filesystem_caches_path.xml.tmp /etc/clickhouse-server1/config.d/filesystem_caches_path.xml
+
+    sudo cat /etc/clickhouse-server2/config.d/filesystem_caches_path.xml \
+    | sed "s|<filesystem_caches_path>/var/lib/clickhouse/filesystem_caches/</filesystem_caches_path>|<filesystem_caches_path>/var/lib/clickhouse/filesystem_caches_1/</filesystem_caches_path>|" \
```
Comment (CheSema) [inline docker/test/stateless/run.sh:78]: Here. clickhouse-server2 also has `/var/lib/clickhouse/filesystem_caches_1`.

Comment (kssenii) [reply docker/test/stateless/run.sh:78]: yes, this is a typo, wanted them to be different

**Review by CheSema — APPROVED** [review]

**Pull request merged**